### PR TITLE
ENTSWM-391: downgraded to Swarm 7.0.1

### PR DESCRIPTION
**Pull request by michalszynkiewicz**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -15,11 +15,14 @@
   <version>7.1.0-redhat-1-SNAPSHOT</version>
   <packaging>pom</packaging>
   <properties>
-    <version.wildfly.swarm>7.1.0.redhat-33</version.wildfly.swarm>
-    <version.infinispan>8.2.8.Final</version.infinispan>
+    <version.wildfly.swarm>7.0.1.redhat-20</version.wildfly.swarm>
+    <version.infinispan>8.1.8.Final-redhat-1</version.infinispan>
```
Comment (michalszynkiewicz) [inline pom.xml]: For some reason, after downgrading to 7.0.1, even the cute-name-service contains infinispan artifacts in the *-swarm.jar.
I had to align Infinispan version to the one used by Swarm.

I double-checked and in 7.1 it is not an issue.
Is it something worth investigating?

Comment (Ladicek) [reply pom.xml]: Well that sounds weird...

Comment (heiko-braun) [reply pom.xml]: Could be related to version mismatches

**Comment by heiko-braun** [comment]
Do we wait for @Ladicek 's review? Or is it good to merge?

**Comment by Ladicek** [comment]
I did take a look yesterday and with the exception of Infinispan version, everything looks fine.

And I don't really understand your (@michalszynkiewicz) comment about Infinispan JARs etc., but I guess you checked this before submitting the PR and it works :-)

**Comment by heiko-braun** [comment]
@Ladicek  It's "ON QE" now anyway :)

**Comment by michalszynkiewicz** [comment]
tested on minishift and locally, works on my machine ;)

**Pull request merged**